metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -132,6 +147,14 @@ def __init__(self, config: KafkaSourceConfig, ctx: PipelineContext):
                 **self.source_config.connection.consumer_config,
             }
         )
+        # TODO: Do we require separate config than existing consumer_config ?
+        self.admin_client = confluent_kafka.admin.AdminClient(
```

Comment:
let's make sure that we catch and warn.. let's not fail the connector..

Reply:
done